src/Microsoft.AspNetCore.SignalR.Client/RpcConnection.cs

```diff
@@ -47,6 +47,41 @@ public void On(string methodName, Type[] parameterTypes, Action<object[]> handle
             _handlers.AddOrUpdate(methodName, invocationHandler, (_, __) => invocationHandler);
         }
 
+        public void On<T1>(string methodName, T1 parameter1, Action<object[]> handler)
```

Comment:
I don't think these should be methods on this class. Rather I'd prefer these to be extension methods.